src/extensions/replay/sessionrecording.ts

```diff
@@ -1270,6 +1282,24 @@ export class SessionRecording {
         logger.info(LOGGER_PREFIX + ' recording resumed')
     }
 
+    private _addEventTriggerListener() {
+        if (this._eventTriggers.length === 0 || !isNullish(this._removeEventTriggerCaptureHook)) {
+            return
+        }
+
+        this._removeEventTriggerCaptureHook = this.instance._addCaptureHook((eventName) => {
```

Comment:
we could effectively inline here and instead of calling the `_addCaptureHook` method

call the more obviously public `on` method that it delegates to

but not blocking